charts/stable/docker-compose/values.yaml

```
@@ -23,25 +23,6 @@ service:
         port: 2376
         type: HTTPS
```

```suggestion
        type: https
```
Fixing this at the same time